lib/resources/config/cluster-config.json

```diff
@@ -0,0 +1,60 @@
+{
+	"Small": {
+		"size" : 3,
+		"master-type" : "m5.xlarge",
+		"core-type" : "m5.xlarge"
+	},
+	"Medium": {
+		"size" : 5,
+		"master-type" : "m5.xlarge",
+		"core-type" : "m5.xlarge"
+	},
+	"Large": {
+		"size" : 7,
+		"master-type" : "m5.xlarge",
+		"core-type" : "p3.8xlarge"
+	},
+	"auto-scaling": {
+		"min-instances" : 2,
+		"max-instances": 15,
+		"scale-out": {
+			"increment": 1,
+			"percentage-mem-available": 15.0,
+			"eval-periods": 1,
+			"cooldown": 300
+		},
+		"scale-in": {
+			"increment": -1,
+			"percentage-mem-available": 75.0,
+			"eval-periods": 1,
+			"cooldown": 300
+		}
+	},
+	"ami" : "CUSTOM_AMI",
+	"release" : "emr-6.2.0",
```

Comment:
This is actually just the existing cluster-config.json. We don't use this release value, but I'm leaving it in there since we want to keep around `cluster-config.json` for one more release while we migrate customers to using app config.